app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+class User < ApplicationRecord
```

Comment:
Constant variable you put into firstly, validate secondly and last is callback (before_save)

Reply:
Done